extensions/podman/src/podman-cli.ts

```diff
@@ -51,7 +51,7 @@ export function getPodmanCli(): string {
 // Get the Podman binary path from configuration podman.binary.path
 // return string or undefined
 export function getCustomBinaryPath(): string | undefined {
-  return extensionApi.configuration.getConfiguration('podman').get('binary.path');
+  return extensionApi.configuration.getConfiguration('podman').get('binary.pathToBinary');
```

Comment:
maybe we need in the extension startup, check if we have the old property name and migrate it to the new name as well ?

Reply:
Hi @benoitf, I have checked references for the old property name, `podman-cli.ts`is the only place I found the reference to `binary.path`.

I believe this feature was introduced in #941. I looked at this  PR and found that `binary.path` was only referenced in `podman-cli-.ts`.

**FYI** - I am fairly new to the codebase and open-source contribution, so please let me know if I have missed anything!

Reply:
so here, probably we would need to check for an old value and then save it to the new value or check both old and new value

Reply:
@aditipatelpro thanks for contributing, I have added some pointers to guide you on how to read older configuration value ( let say a user is migrating from an old version of Podman Desktop to a newer version where the property is now updated)

Reply:
@benoitf, thanks for guiding. Let me look at the pointers you suggested. Thanks again.